spec/lib/models/concerns/has_audit_trail_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "spec_helper"
+
+module Timelines
```

Comment:
I wasn't able to run this test, or any of the tests in this gem. If I `cd` into the `timelines` folder and run `rspec`, I get this error from ` DatabaseCleaner.clean`:

```
 ActiveRecord::NoDatabaseError:
              We could not find your database: timelines_dummy_test. Available database configurations can be found in config/database.yml.
            
              To resolve this error:
            
              - Did you not create the database, or did you delete it? To create the database, run:
            
                  bin/rails db:create
```

I tried running `rails db:create` but it just tells me that `Database 'devlin_development' already exists`, so it's trying to perform that on the main repo rather than the local gem.

Reply:
Could you try running `RAILS_ENV=test bundle exec rake db:create`?

Reply:
I ran this and got the following error:
```
Don't know how to build task 'db:create' (See the list of available tasks with `rake --tasks`)
```
How have you been running tests locally? Do they just work with `bundle exec rspec`?

Reply:
Ah, looks like you need to `cd spec/dummy` to run the database setup commands. Then you can run `rspec` from the project root